src/plugins/telemetry/server/telemetry_collection/get_cluster_info.ts

```diff
@@ -35,14 +35,36 @@ export interface ESClusterInfo {
     minimum_index_compatibility_version: string;
   };
 }
-
+export interface ESClusterInfoResponse {
+  body: ESClusterInfo;
+  statusCode: number;
+  headers: object;
+  warnings: string[];
+  meta: object;
+}
+export async function clusterInfoGetter(esClient: ElasticsearchClient) {
+  const { body } = ((await esClient.info()) as unknown) as ESClusterInfoResponse;
```

Comment:
```suggestion
  const { body } = await esClient.info<ESClusterInfo>();
```

and no need for `ESClusterInfoResponse` 😉